e2e/react-start/flamegraph-bench/src/routes/page.$id.tsx

```
@@ -0,0 +1,29 @@
+import { createFileRoute, Link } from '@tanstack/react-router'
```

ESLint

[error] Member 'Link' of the import declaration should be sorted alphabetically.

(sort-imports)